portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -60,4 +60,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     datastore.put(commentEntity);
     doGet(request, response);
   }
+
+  @Override
+  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment").setKeysOnly();
```

Comment:
What does `setKeysOnly` do

Reply:
it returns only the keys of the entities!